app/lib/dartdoc/backend.dart

```diff
@@ -434,4 +436,14 @@ class DartdocBackend {
     sw.stop();
     _logger.info('$prefix: $count files deleted in ${sw.elapsed}.');
   }
+
+  Future _deleteFile(String path) async {
+    try {
+      await _storage.delete(path);
+    } on DetailedApiRequestError catch (e, _) {
```

Comment:
} on DetailedApiRequestError catch (e) {

Or is the _ needed to keep the stack trace for the rethrow?

Reply:
I thought it is needed, but experimenting with a mock code, it seems it is not necessary, removing it...